packages/flutter_tools/lib/src/commands/build_aar.dart

```diff
@@ -99,9 +99,8 @@ class BuildAarCommand extends BuildSubCommand {
   final String description = 'Build a repository containing an AAR and a POM file.\n\n'
       'By default, AARs are built for `release`, `debug` and `profile`.\n'
       'The POM file is used to include the dependencies that the AAR was compiled against.\n'
-      'To learn more about how to use these artifacts, see '
-      'https://flutter.dev/go/build-aar\n'
-      'Note: this command builds applications assuming that the entrypoint is lib/main.dart. '
+      'To learn more about how to use these artifacts, see https://flutter.dev/go/build-aar\n'
+      'This command builds applications assuming that the entrypoint is lib/main.dart. '
       'This cannot currently be configured.';
```

Comment:
```suggestion
      'This cannot be configured currently.';
```

Reply:
This is one of those cases where the earlier adverb acts as emphasis, I think.

Reply:
Feel free to ignore this, I don't feel strongly :)